bin/setup

```diff
@@ -6,36 +6,41 @@
 # Exit if any subcommand fails
 set -e
 
-# Set up Ruby dependencies via Bundler
-gem list bundler --installed > /dev/null || gem install bundler
-bundle install
+if [ -z "$1" ]; then
```

Comment:
This seems like a lot of work to require each time. How about instead we add a line that recommends that the user add their own. If they want to remove that line, they're welcome to do so and stop being bugged about it.